ha/core/controllers/pcs/node_controller.py

```diff
@@ -114,7 +119,29 @@ def start(self, nodeid: str) -> dict:
             ([dict]): Return dictionary. {"status": "", "msg":""}
                 status: Succeeded, Failed, InProgress
         """
-        raise HAUnimplemented("This operation is not implemented.")
+        _res = self.nodes_status([nodeid])
+        _node_status = _res.get(nodeid)
+        if _node_status.lower() == NODE_STATUSES.ONLINE.value.lower():
+            return {"status": "Succeeded", "msg": f"Node {nodeid}, is already in Online status"}
+        elif _node_status.lower() == NODE_STATUSES.STANDBY_WITH_RESOURCES_RUNNING.value.lower():
+            return {"status": "Succeeded", "msg": f"Node {nodeid}, is going in standby mode, "
+                                                  f"We need to wait to complete the resource shutdown"}
+        elif _node_status.lower() == NODE_STATUSES.STANDBY.value.lower():
+            # make node unstandby
+            _output, _err, _rc = self._execute.run_cmd(const.PCS_NODE_UNSTANDBY.replace("<node>", nodeid),
+                                                       check_error=False)
+            if self.check_resource_failcount():
+                Log.error(f"Node {nodeid} : Resource failcount found in the node start operation")
+                return {"status": "Failed", "msg": f"Node {nodeid} : Resource failcount found "
+                                                   f"in the node start operation"}
+            else:
+                return {"status": "Succeeded", "msg": f"Node {nodeid} : Node was in standby mode, "
```

Comment:
{"status": "Succeeded",


status should be InProgress as resource not started yet

Reply:
Done